### ISSUE TITLE
Funding request system needed

### ISSUE THREAD

**Issue by stickster** [puiterwijk/regcfp#198]
Sometimes attendees may have a hard time coming up with a registration fee. They need to be able to indicate why they may need help with the fee. There should be an option for indicating a hardship and a justification (which remains visible only to the registrant and those handling fees). We need this for Flock 2017.

**Comment by puiterwijk** [comment]
Would this require any special handling? Or could we just use a standard text field for this?

**Comment by stickster** [comment]
@mattdm was describing to me more of a worksheet approach. Tagging him for some additional commentary.

**Comment by mattdm** [comment]
The approach we have previously used can be caricatured as a checkbox which says "Yes, I like free money!". We want to move away from that. :)

Specifically, we want:

* To encourage people to think of this as financial assistance, not a giveaway
* To allow people to specify less than 100% (and to encourage that)
* For people to let us know if funding is really essential to their attendance.

I'm thinking of two expanding/dropdown areas on the form; the first being:


* Click here to help sponsor another contributors' attendance. Any amount helps!
  * $5 / $25 / $50 / $ ____

* Click here if you will be unable to attend Flock without financial assistance.
  * Enter the amount of aid you need in each area:
    * Registration fee ___
    * Transportation  ___
    * Lodging ___
    * Meals ____
  * In order to attend Flock, I would like Fedora to cover __ 80% __ of this cost. [ $calculated ]
  * Paragraph here that we try our best to get every contributor who needs to be there to Flock but funds are limited and not all requests can be accommodated.

**Comment by bexelbie** [comment]
@mattdm my recollection is that we didn't want to ask people to justify their funding requests.  We wanted instead to ask all attendees why they were attending flock and how they though their attendance would help them and the project.

We do not want to be in the business of adjudicating need.

**Comment by mattdm** [comment]
I agree that we don't want to be in that business, and that we want to make decisions primarily on who we need to be there rather than need per se. But I definitely I don't want to give people the impression that just because you ask, you'll be funded.

**Comment by bexelbie** [comment]
@mattdm I think between asking everyone why they are coming and language on the funding page we can make this clear.
